external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HdfsSpout.java

```diff
@@ -221,14 +221,18 @@ public void nextTuple() {
         while (true) {
             try {
                 // 3) Select a new file if one is not open already
+                boolean newReader = false;
                 if (reader == null) {
                     reader = pickNextFile();
                     if (reader == null) {
                         LOG.debug("Currently no new files to process under : " + sourceDirPath);
                         return;
                     } else {
                         fileReadCompletely = false;
+                        newReader = true;
                     }
+                } else {
+                    newReader = false;
```

Comment:
Nit: This branch is a no op, newReader is already false

Reply:
Thank you for pointing this out!